lib/model/imapmessage.php

```diff
@@ -479,6 +479,9 @@ public function getFullMessage($ownMail, $specialRole=null) {
 		$data = $this->getListArray();
 		if ($this->hasHtmlMessage) {
 			$data['hasHtmlBody'] = true;
+			if($specialRole === 'drafts'){
+				$data['body'] = rtrim($this->htmlMessage,"<br><br>");
```

Comment:
(not that github hides/replace some html tags like ``<br>``, make sure you escape those)